tests/saw/Makefile

```diff
@@ -245,6 +245,12 @@ BIKE_R1_COPY = $(addprefix s2n/pq-crypto/bike_r1/, $(notdir $(BIKE_R1_C)))
 BIKE_R2_C = $(wildcard ../../pq-crypto/bike_r2/*.c) $(wildcard ../../pq-crypto/bike_r2/*.h) ../../pq-crypto/bike_r2/Makefile
 BIKE_R2_COPY = $(addprefix s2n/pq-crypto/bike_r2/, $(notdir $(BIKE_R2_C)))
 
+#The scripts below for correcting the failed SAW buildings. 
+#It is not to run any SAW tests on Kyber code
+#It is just to make building without failure during the BIKE/SIKE SAW tests
+KYBER_R2_C = $(wildcard ../../pq-crypto/kyber_r2/*.c) $(wildcard ../../pq-crypto/kyber_r2/*.h) ../../pq-crypto/kyber_r2/Makefile
+KYBER_R2_COPY = $(addprefix s2n/pq-crypto/kyber_r2/, $(notdir $(KYBER_R2_C)))
```

Comment:
You'll also need to add a target to create the directory for copying, similar to https://github.com/awslabs/s2n/blob/52c59b87d3c6a548c11caf5d279f5ac2ee50083d/tests/saw/Makefile#L286-L287

(That was my mistake for not mentioning that in my previous comment.)

Reply:
Sorry, I missed another spot that needs to be updated: 

https://github.com/awslabs/s2n/blob/7c232812a26c29c4935a70deecb4bf3ef84879ae/tests/saw/Makefile#L344

Update that target to include `s2n/pq-crypto-kyber_r2` and `$(KYBER_R2_COPY)`. I *think* it will build correctly then.

Reply:
I followed your comments. It works now.